code/__HELPERS/hallucinations.dm

```diff
@@ -0,0 +1,229 @@
+/// A global list of all ongoing hallucinations, primarily for easy access to be able to stop (delete) hallucinations.
+GLOBAL_LIST_EMPTY(all_ongoing_hallucinations)
+
+/// Wrapper for _cause_hallucination to use named arguments
+#define cause_hallucination(arguments...) _cause_hallucination(list(##arguments))
+
+/// Causes a hallucination of a certain type to the mob.
+/// Use the wrapper for named argument support, don't use this.
```

Comment:
This documentation should be on `cause_hallucination`, since it is what will actually show on hover.

The "Wrapper for _cause_hallucination to use named arguments" text should be a normal comment because it's not really important for documentation, just implementation.